.circleci/config.yml

```diff
@@ -40,7 +40,7 @@ jobs:
                   command: composer install --no-scripts
             - run:
                   name: Create .env
-                  command: cp .env.dist .env
+                  command: cp .env
```

Comment:
Does not make sens (and does not work I guess)

Reply:
Yes I completely removed it later; it still doesn't work though :stuck_out_tongue: